### ISSUE TITLE
options.contents extraction for cssauron returns array, is it meant to be a string?

### ISSUE THREAD

**Issue by leiyangyou** [StephanHoyer/mithril-query#18]
```javascript
    return isArray(node.children) ? node.children.filter(isString).concat('') : '';
```

may need to be

```javascript
    return isArray(node.children) ? node.children.filter(isString).join('') : '';
```

**Comment by StephanHoyer** [comment]
you're right... feel free to open a PR

**Comment by StephanHoyer** [comment]
fixed in current release 0.7.8
